### ISSUE TITLE
Dataset preperation for SWIN-IR

### ISSUE THREAD

**Issue by yossigandelsman** [cszn/KAIR#138]
Hi, I am trying to recreate the results from the paper. 
I downloaded the two training datasets for "classical/lightweight image SR" from the provided links in [your documentation](https://github.com/JingyunLiang/SwinIR#Training).
Unfortunately, I only see 743 images in the DIV2K_train_HR directory under DIV2K and not 800. Did you also use the test set? 
In the Flickr2K I only see the LR versions (Flickr2K_LR_bicubic, Flickr2K_LR_unknown). Should I download the HR images from somewhere else?

Thanks for the help!

**Comment by JingyunLiang** [comment]
I think your file may be corrupted. I downloaded it just now can find there are 900 images in DIV2K. You should use the first 800 images.
